.travis.yml

```diff
@@ -61,6 +61,4 @@ matrix:
             - gcc-4.9
             - gcc-4.9-multilib
       env: OPTS="--test=host-linux --buildtype=debug --buildoptions=--target-arch=i686,--compile-flag=-fsanitize=undefined,--compile-flag=-fno-omit-frame-pointer,--compile-flag=-fno-common,--jerry-cmake-param=-DJERRY_LIBC=OFF,--jerry-cmake-param=-DFEATURE_SYSTEM_ALLOCATOR=ON,--no-snapshot,--no-check-valgrind" INSTALL_TRAVIS_I686_DEPS=yes UBSAN_OPTIONS=print_stacktrace=1
-  allow_failures:
```

Comment:
Yes, it's intentional.
The first task is an address-sanitizer (`--compile-flag=-fsanitize=address`), the second one is an undefined-behaviour sanitizer (`--compile-flag=-fsanitize=undefined`).